setup.py

```diff
@@ -29,10 +31,13 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
     ],
-    packages=['pip_missing_reqs'],
+    packages=['pip_check_reqs'],
     entry_points={
         'console_scripts': [
-            'pip-missing-reqs=pip_missing_reqs.find_missing_reqs:main',
+            'pip-missing-reqs=pip_check_reqs.find_missing_reqs:main',
+        ],
+        'console_scripts': [
+            'pip-extra-reqs=pip_check_reqs.find_extra_reqs:main',
         ],
```

Comment:
You're defining console_scripts twice - the second will supplant the first. Should just be once, with two-element list.